src/compilationSamples/helpers/helpers.ts

```diff
@@ -15,6 +15,7 @@ export const defaultCompilerOptions: ts.CompilerOptions = {
   target: ts.ScriptTarget.ES2016,
   module: ts.ModuleKind.CommonJS,
   jsxFactory: 'react',
+  esModuleInterop: true,
```

Comment:
What is this for? From my glance at the docs this would affect the TS output, hence I am not sure why we need it in the test (which doesn't output anything for later consumption).

Reply:
This was needed because the way `jest-diff` exports the functions. I think this is actually only being used for typing and has no effect on code generation.